src/sass/app.scss

```diff
@@ -269,6 +266,10 @@ Footer a{
     align-content: center;
 }
 
+.space-between {
```

Comment:
I think we can just use bootstrap for this
https://getbootstrap.com/docs/4.0/utilities/flex/#justify-content

Reply:
I see done!